src/vivarium/examples/disease_model/mortality.py

```diff
@@ -49,11 +46,9 @@ def setup(self, builder: Builder) -> None:
         builder
             Access to simulation tools and subsystems.
         """
-        self.config = builder.configuration.mortality
         self.randomness = builder.randomness.get_stream("mortality")
-
-        builder.value.register_rate_producer(
-            "mortality_rate", source=self.base_mortality_rate, component=self
+        self.mortality_rate = builder.value.register_rate_producer(
```

Comment:
Should you not assign it since we know we'll have to change it anyway w/ the pop refactor?